presto-docs/src/main/sphinx/optimizer/join-distribution-type.rst

```diff
@@ -5,33 +5,52 @@ Repartitioned and Replicated Joins
 Background
 ----------
 
-Presto can perform two types of distributed joins: repartitioned and replicated. In a repartitioned join, both inputs to a
```

Comment:
use ===== instead of ***** for the main heading, check other docs to have consistency,